modules/contracts/src.sol/CMCCore.sol

```diff
@@ -66,7 +66,7 @@ contract CMCCore is ReentrancyGuard, ICMCCore {
     }
 
     /// @notice A getter function for the bob of the multisig
-    /// @return Alice's signer address
+    /// @return Bob's signer address
```

Comment:
@LayneHaber @rhlsthrm idk when this happened, probably when I was snooping around codebase and noticed this weirdness. Lmk if you want me to remove as irrelevant to PR... but also it's just a comment haha

Reply:
nvm removed